website/docs/src/markdown-conversion/unified-plugins/rehype-label.js

```diff
@@ -0,0 +1,34 @@
+import { visit } from 'unist-util-visit';
```

Comment:
Could you document at the top in short: What this file does and what its used for very shortly. Had to review one of these, but was written 4 years ago. Makes it easier when documented